app/components/provider_interface/organisation_list_component.rb

```diff
@@ -0,0 +1,22 @@
+module ProviderInterface
+  class OrganisationListComponent < ViewComponent::Base
+    attr_accessor :training_permissions, :ratifying_permissions, :current_provider_user
+
+    def initialize(provider:,  current_provider_user:)
+      @current_provider_user = current_provider_user
+      @training_permissions = ProviderRelationshipPermissions.where(training_provider: provider)
+        .or(ProviderRelationshipPermissions.where(training_provider: all_manageable_providers, ratifying_provider: provider))
+        .where.not(setup_at: nil)
+        .includes(:training_provider)
+      @ratifying_permissions = ProviderRelationshipPermissions.where(ratifying_provider: provider.id)
+        .where.not(training_provider: all_manageable_providers)
```

Comment:
The ratifying provider list here is a read only version of ratifying providers permissions which a user does not have manage permissions access to. We have attempted to keep the previous logic the same (although it will be changing in the work coming soon) however we amended the query slightly to avoid diffing the training permissions `- @training_permissions` (from the previous implementation)  and instead found that using a `where.not` query will yield similar results.